### DISCUSSION TITLE
.NET 9 Release?

### DISCUSSION THREAD

**Discussion by Meaders123** [enisn/UraniumUI#832] [Q&A] [answered]
Hi guys, do we have an estimated release date on a .NET 9? Or is that way in the future? 👍

**Comment by enisn** [comment, marked as answer]
Currently .NET 9 is supported with preview release which is `v2.11.0-pre.1` you can find it on the NuGet from [here](https://www.nuget.org/packages/UraniumUI/2.11.0-pre.1#supportedframeworks-body-tab)

The stable version will be published soon
